_pages/people/lguerard.md

```diff
@@ -0,0 +1,18 @@
+---
+title: lguerard
+name: Laurent Guerard
+affiliation:
+  - IMCF University of Basel | https://www.biozentrum.unibas.ch/research/researchgroups/overview/unit/imcf/imaging-core-facility-%20imcf/
+forum: lguerard
+github: lguerard
+twitter: @lguerard42
+linkedin: www.linkedin.com/in/laurent-guerard-360b2486
```

Comment:
This should be simply `in/laurent-guerard-360b2486`, without the `www.linkedin.com`.